pkg/builds/build_info.go

```diff
@@ -74,7 +74,7 @@ func CreateBuildPodInfo(pod *corev1.Pod) *BuildPodInfo {
 	containers, _, isInit := kube.GetContainersWithStatusAndIsInit(pod)
 
 	for _, container := range containers {
-		if strings.HasPrefix(container.Name, "build-step-git-source") {
+		if strings.HasPrefix(container.Name, "build-step-git-source") || strings.HasPrefix(container.Name, "step-git-source") {
```

Comment:
Wouldn't the second condition also be true for the first one? the first one contains "step-git-source".

Reply:
I thought `HasPrefix` looked for, in regex terms, `/^build-step-git-source.*/`?

Reply:
Nvm me! it's HasPrefix and not Contains :D :D :D

Reply:
=) I actually considered going with `Contains` but decided to make sure we didn't get confused by someone actually naming a step something like `Step Git Source`, which would end up as a container named `step-step-git-source`, and wouldn't be the right container to look at, etc, etc...